plugins/google-sheets/src/pages/Authenticate.tsx

```diff
@@ -56,12 +65,25 @@ export function Authenticate({ onAuthenticated }: AuthenticationProps) {
     return (
         <div className="col-lg pb-[15px]">
             <Hero />
-            <ol className="list-decimal list-inside space-y-2.5 marker:text-secondary *:text-tertiary *:leading-none *:tracking-normal py-[7px]">
-                <li>Log in to your Google account</li>
-                <li>Ensure your sheet has a header row</li>
-                <li>Map the column fields to the CMS</li>
+            <ol className="list-decimal list-inside space-y-2.5 marker:text-secondary *:text-content *:leading-none *:tracking-normal py-[7px]">
+                <li>
+                    <span className="pl-[10px]">Log in to your Google account</span>
```

Comment:
I can't believe its this annoying to style the list marker but unfortunately with `    list-style-position: inside;` you don't have control over the padding of the text after the marker.

And using outside would require some hardcoded value which could differ between browsers.

https://css-tricks.com/everything-you-need-to-know-about-the-gap-after-the-list-marker/